_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,97 @@
+detailing Fortran news from the previous month.
+
+* [This website](#this-website)
+* [Standard Library](#standard-library)
+* [Package Manager](#package-manager)
+* [Events](#events)
+
+## This website
+
+If you came to this newsletter from elsewhere, welcome to the new Fortran website.
+We built this site mid-April and hope for it to be _the_ home of Fortran on the internet,
+which traditionally there hasn't been any to date.
+Look around and [let us know](https://github.com/fortran-lang/fortran-lang.github.io/issues)
+if you have any suggestions for improvement.
+Specifically, [Learn](/learn) and [Packages](/packages) are the pages that
+we'll be focusing on in the coming months.
+Please help us make them better!
+
+* [#90](https://github.com/fortran-lang/fortran-lang.org/pull/90)
+WIP: Mini-book on building programs
+
+## Fortran Standard Library
+
+Here's what's new in Fortran Standard Library:
+
+* [#191](https://github.com/fortran-lang/stdlib/pull/191)
+WIP: Function for computing Pearson correlations among elements of
+an array in the `stdlib_experimental_stats` module.
+
+* [#189](https://github.com/fortran-lang/stdlib/pull/189)
+WIP: Procedures for sparse matrices operations. Ongoing discussion on the API can be found
+[here](https://github.com/fortran-lang/stdlib/wiki/Stdlib-Sparse-matrix-API)
+
+* [#183](https://github.com/fortran-lang/stdlib/pull/183)
+Automatic API-doc generation and deployment of this [stdlib website](https://stdlib.fortran-lang.org)
+
+* [#170](https://github.com/fortran-lang/stdlib/pull/170)
+Addition of the new functions `diag`, `eye`, and `trace` functions to make working with
+matrices easier.
+Read the full specifications [here](https://stdlib.fortran-lang.org/page/specs/stdlib_experimental_linalg.html)
+
+## Package Manager
+
+In the past months we've seen the first working implementation of the [Fortran Package Manager (fpm)](https://github.com/fortran-lang/fpm).
+Specifically:
+
+* fpm supports three commands:
+  - `fpm build`--compiles and links your application and/or library.
+  - `fpm test`--runs tests if your package has any test programs.
+  - `fpm run`--runs the application if your package has an executable program.
+* fpm can build an executable program, a library, or a combination of both.
+* fpm can fetch and compile other Fortran libraries using fpm that your project depends on
+* Currently only gfortran is supported as the compiler backend. fpm will suport other compilers soon.
+
+Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md)
+to learn how to build your package with fpm.
+
+fpm is still in very early development, and we need as much help as we can get.
+Here's how you can help today:
+
+* Try to use it. Does it work? No? Let us know!
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm.
+* Improve the documentation.
```

Comment:
But a call to action like this I think it's okay to repeat because we need help.